optoforce/config/SR_OMD45FE1000N.yaml

```diff
@@ -0,0 +1,6 @@
+counts_n_fx: 71.07
```

Comment:
Another one called `multi_channel_3_axis_generic_scale.yaml` could be added, as the default value in hte launchfile is "m-ch/3-axis".